### PR TITLE
Updated Unity Editor 2017.2.0f3 MRTP3

### PR DESCRIPTION
**Pull request by StephenHodgson**
Fixes: #1245 

Preview:
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,19 +14,18 @@ MixedRealityToolkit-Unity uses code from the base [MixedRealityToolkit](https://
 > Learn more about [Windows Mixed Reality](https://www.microsoft.com/en-gb/windows/windows-mixed-reality) here
 
 [github-rel]:                  https://github.com/Microsoft/MixedRealityToolkit-Unity/releases/latest
-[github-rel-badge]:            https://img.shields.io/github/tag/microsoft/MixedRealityToolkit-unity.svg?style=flat-square&label=latest%20master%20branch%20release&colorB=007ec6
```
Comment (keveleigh) [inline README.md]: We should keep this automatic badge updating URL in the README.

File: README.md [hunk 2]
```diff
@@ -14,19 +14,18 @@ MixedRealityToolkit-Unity uses code from the base [MixedRealityToolkit](https://
...
 
 [githubhl-rel]:                  https://github.com/Microsoft/MixedRealityToolkit-Unity/releases/latest
-[mrtk-version-badge]:            https://img.shields.io/badge/HoloLens%20MRTK%20release-2017.1.2-blue.svg
+[mrtk-version-badge]:            https://img.shields.io/badge/HoloLens%20MRTK%20Release-%20v1.2017.2.0-blue.svg
 
-[unity-release]:
-https://unity3d.com/unity/whats-new/unity-2017.1.2
-[githubhl-rel-badge]:            https://img.shields.io/badge/Unity%20HoloLens%20release-2017.1.2-blue.svg
+[unity-release]:                 https://unity3d.com/unity/whats-new/unity-2017.1.2
+[githubhl-rel-badge]:            https://img.shields.io/badge/Unity%20Editor%20HoloLens%20Release-2017.1.2.0f3-blue.svg
```
Comment (keveleigh) [inline README.md]: Why is the [v1.2017.2.0](https://github.com/Microsoft/MixedRealityToolkit-Unity/releases/tag/v1.2017.2.0) release marked for use with Unity 2017.1.2? This should be v1.2017.1.3 instead (or v1.2017.1.2.1 if we really want that many numbers haha).

v1.2017.2.0 should work with Unity 2017.2, per the [roadmap](https://github.com/Microsoft/MixedRealityToolkit-Unity#roadmaps).

File: README.md (hunk 1 above)
Comment (StephenHodgson) [reply README.md]: We _should_ but for this instance of the release we want to point people to the "WMR" flavor for this once

Comment (keveleigh) [reply README.md]: True, but the HoloLens one should still point to the "latest", right?

File: README.md (hunk 2 above)
Comment (StephenHodgson) [reply README.md]: Generally I try to keep the release tags as close to Unity's as possible.

I actually had matching the release but had to backtrack when we found out that the latest 2017.2 is incompatible with HoloLens building using the later SDK targets.

File: README.md (hunk 1 above)
Comment (StephenHodgson) [reply README.md]: Good point

Comment (StephenHodgson) [reply README.md]: Updated the HoloLens section to use the latest.

**Pull request merged**